spec/controllers/api/v1/passwords_controller_spec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe API::V1::PasswordsController, type: :controller do
```

Comment:
Why dont we use `request` type as `type: :controller` is [obsolete](http://rspec.info/blog/2016/07/rspec-3-5-has-been-released/) now
```suggestion
RSpec.describe API::V1::PasswordsController, type: :request do
```